src/diagnostics/diagnostic.jl

```diff
@@ -150,6 +150,19 @@ function diagnostic_as_vectors(writer::DictWriter, diagnostic; layer = 1)
     return times, vector_layer_n
 end
 
+"""
+    close_diagnostics(sim)
+
+Close the output writers in the diagnostics.
```

Comment:
I wonder if this should take `diagnostics` as the argument. Then it can be used also in cases where we dont set up a land simulation.

and then in the other function, we can call close_output_writers(sim.diagnostics).